.github/APOLLO_RELEASE_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+Release @apollo/PACKAGE1@X.Y.Z [, @apollo/PACKAGE2@X.Y.Z]
+
+As with [release PRs in the past](https://github.com/apollographql/federation/issues?q=label%3A%22:label:%20release%22+is%3Aclosed), this is a PR tracking a `release-x.y.z` branch for an upcoming release. 🙌   The version in the title of this PR should correspond to the appropriate branch.
+
+Check the appropriate milestone (to the right) for more details on what we hope to get into this release!
+
```

Comment:
Oh, I lifted this [from Apollo Server's template](https://github.com/apollographql/apollo-server/blob/main/.github/APOLLO_RELEASE_TEMPLATE.md), but I realize this isn't always true.

I suggest removing my own inclusion of it!

```suggestion
```